azure-quantum/azure/quantum/target/microsoft/elements/dft/target.py

```diff
@@ -73,15 +75,153 @@ def submit(self,
         if shots is not None:
             warnings.warn("The 'shots' parameter is ignored in Microsoft Elements Dft job.")
         
-        return super().submit(
-            input_data=input_data,
-            name=name, 
-            shots=shots, 
-            input_params=input_params,
-            **kwargs
-        )
+        if isinstance(input_data, list):
+
+            qcschema_data = self._assemble_qcshema_from_files(input_data, input_params)
+
+            qcschema_blobs = {}
+            for i in range(len(qcschema_data)):
+                qcschema_blobs[f"inputData_{i}"] = self._encode_input_data(qcschema_data[i])
 
+            toc_str = self._create_table_of_contents(input_data, list(qcschema_blobs.keys()))
+            toc = self._encode_input_data(toc_str)
+
+            return self._get_job_class().from_input_data_container(
+                workspace=self.workspace,
+                name=name,
+                target=self.name,
+                input_data=toc,
+                batch_input_blobs=qcschema_blobs,
+                input_params={ 'numberOfFiles': len(qcschema_data), "inputFiles": list(qcschema_blobs.keys()), **input_params },
+                content_type=kwargs.pop('content_type', self.content_type),
+                encoding=kwargs.pop('encoding', self.encoding),
+                provider_id=self.provider_id,
+                input_data_format=kwargs.pop('input_data_format', 'microsoft.qc-schema.v1'),
+                output_data_format=kwargs.pop('output_data_format', self.output_data_format),
+                session_id=self.get_latest_session_id(),
+                **kwargs
+            )
+        else:
+            return super().submit(
+                input_data=input_data,
+                name=name, 
+                shots=shots, 
+                input_params=input_params,
+                **kwargs
+            )
+
+
+    
+    @classmethod
+    def _assemble_qcshema_from_files(self, input_data: List[str], input_params: Dict) -> str:
+        """
+        Convert a list of files to a list of qcshema objects serialized in json.
+        """
+
+        qcshema_objects = []
+        for file in input_data:
+            file_path = Path(file)
+            if not file_path.exists():
+                raise FileNotFoundError(f"File {file} does not exist.")
+            
+            file_data = file_path.read_text()
+            mol = self._xyz_to_qcschema_mol(file_data)
```

Comment:
Should we handle the case where provided input files are already a `QCSchema`? In such case, we will just add them to `qcschema_objects` without transformation. We could naively assume format from the file extension. 
There could also be a mix of `.xyz` and `.qcschema` files provided to the job, so handling them in this loop still makes sense.

Reply:
Done

Reply:
I used `.json` as the file extension for qcschema. I also added a warning in case they provide both input parameters and a qcschema. It will use the parameters in the qcschema rather than modifying it with the input parameters.